### PR TITLE
Fix blog errors

### PR DESCRIPTION
**Pull request by sjoerdbeentjes**
Ticket: https://issuetracker.deltares.nl/browse/GWW-194

### CODE REVIEW

File: src/components/CardItem/CardItem.vue [hunk 1]
```diff
@@ -25,10 +25,14 @@
     </p>
 
     <address class="card-item__author">
-      <ResponsiveImage class="card-item__author-image" :image="author.image" />
+      <ResponsiveImage
+        v-if="author"
```
Comment (vvandijck) [inline src/components/CardItem/CardItem.vue:29]: I think this needs to be `v-if="author.image"`. Right?

File: src/components/CardItem/CardItem.vue [hunk 2]
```diff
@@ -25,10 +25,14 @@
...
+        class="card-item__author-image"
+        :image="author.image"
+      />
 
       <div>
-        <span class="p small bold">
+        <span v-if="author" class="p small bold">
```
Comment (vvandijck) [inline src/components/CardItem/CardItem.vue:35]: And this can be `v-if="author.name"` I think

File: src/components/CardItem/CardItem.vue [hunk 3]
```diff
@@ -25,10 +25,14 @@
     </p>
 
     <address class="card-item__author">
-      <ResponsiveImage class="card-item__author-image" :image="author.image" />
+      <ResponsiveImage
+        v-if="author && author.image"
```
Comment (sjoerdbeentjes) [inline src/components/CardItem/CardItem.vue:29]: `author?.image` syntax not available in template, so I used `&&`

File: src/components/CardItem/CardItem.vue [hunk 4]
```diff
@@ -25,10 +25,14 @@
...
+        class="card-item__author-image"
+        :image="author.image"
+      />
 
       <div>
-        <span class="p small bold">
+        <span v-if="author && author.image" class="p small bold">
```
Comment (sjoerdbeentjes) [inline src/components/CardItem/CardItem.vue:35]: `author?.image` syntax not available in template, so I used `&&`

**Review by vvandijck — APPROVED** [review]

**Pull request merged**